### PR TITLE
run ember-native-class-codemod

### PR DESCRIPTION
**Pull request by mcfiredrill**
I ran the codemod to convert to native classes from ember classes.
Originally I got an error running it but I filed this issue and got help:
https://github.com/ember-codemods/ember-native-class-codemod/issues/222

```
ember test --server
npx ember-native-class-codemod http://localhost:7357/502125111151582/tests/index.html app/**/*.js
```

https://github.com/ember-codemods/ember-native-class-codemod

So I just need to check that nothing broke, everything seems fine so far.

### CODE REVIEW

**Comment by bradwbradw** [comment]
so, this is how to run tests? How is the long number in the url determined? (`/502125111151582`)

**Comment by mcfiredrill** [comment]
> so, this is how to run tests? How is the long number in the url determined? (/502125111151582)

Ah sorry, I was just noting the command I used to run the codemod.

File: app/components/chat-message.js [hunk 1]
```diff
@@ -1,20 +1,28 @@
-import Component from '@ember/component';
+import classic from 'ember-classic-decorator';
```
Comment (mcfiredrill) [inline app/components/chat-message.js]: more info about classic decorator
https://github.com/emberjs/ember-classic-decorator#why-do-i-need-this-decorator

**Comment by mcfiredrill** [comment]
Ran again since this was getting so outdated. https://github.com/datafruits/datafruits/pull/372

**Pull request closed without merging**